app/ui/promo.js

```diff
@@ -24,11 +24,11 @@ class Promo extends Component {
             alt="Firefox"
           />
           <span class="ml-2 sm:ml-4 text-xs sm:text-base">
-            ${this.state.translate('trailheadPromo')}${' '}
+            ${`Like Firefox Send? Youʼll love our new VPN.`}
```

Comment:
@johngruen what about "Like Firefox Send? You'll love our new full-device VPN." -- just thought is is important to make sure they know this is a full-device product as anything with "Firefox" immediately gets associated in the consumer mind with browser-only or firefox-only.

Reply:
👍